go/test/endtoend/recovery/recovery_util.go

```diff
@@ -49,14 +49,19 @@ func VerifyQueriesUsingVtgate(t *testing.T, session *vtgateconn.VTGateSession, q
 
 func RestoreTablet(t *testing.T, localCluster *cluster.LocalProcessCluster, tablet *cluster.Vttablet, restoreKSName string, shardName string, keyspaceName string, commonTabletArg []string) {
 	tablet.ValidateTabletRestart(t)
-	tm := time.Now().UTC()
-	tm.Format(time.RFC3339)
-	_, err := localCluster.VtctlProcess.ExecuteCommandWithOutput("CreateKeyspace",
-		"-keyspace_type=SNAPSHOT", "-base_keyspace="+keyspaceName,
-		"-snapshot_time", tm.Format(time.RFC3339), restoreKSName)
-	require.Nil(t, err)
-
 	replicaTabletArgs := commonTabletArg
+
```

Comment:
@zmagg  and I discussed this. It seems like this was a flaky test. The following change addresses that problem.